### PR TITLE
Version 0.9.7

### PR DESCRIPTION
**Pull request by neilalexander**
Changelog and version bump.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## Dendrite 0.9.7 (2022-09-09)
+
+### Features
+
+* Initial supporting code to enable full-text search has been merged (although not ready for use yet)
+* Newly created rooms now have higher default power levels for enabling encryption, setting server ACLs or sending tombstone events
```
Comment (kegsay) [inline CHANGES.md:8]: xref spec?

Comment (neilalexander) [reply CHANGES.md:8]: I don't think it's in the spec, but these three things probably shouldn't be PL 50 and Synapse sets them to PL 100 as default too.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+
+### Fixes
+
+* A race condition between the roomserver output events being generated, forward extremities being updated and room info being updated has been fixed
+* Appservices will no longer receive invite events which they are not interested in, which caused heavy load in some cases or excessive request sizes in others
+* A bug in state resolution v2 where events could incorrectly be classified as control events has been fixed
+* A bug in state resolution v2 where some specific events with unexpected non-empty state keys are dropped has been fixed
+* A bug in state resolution v2 when fetching auth events vs partial state has been fixed
+* Stale device lists should now be handled correctly for all user IDs, which may help with E2EE reliability
+* A number of database writer issues have been fixed in the user API and sync API, which should help to reduce `database is locked` errors (particularly with SQLite)
```
Comment (kegsay) [inline CHANGES.md:19]: Not "particularly" - _only_ in SQLite surely?

**Review by kegsay — APPROVED** [review]

File: CHANGES.md (hunk 2 above)
Comment (neilalexander) [reply CHANGES.md:19]: I suppose so!

**Pull request merged**